games/d.yaml

```
@@ -617,6 +617,28 @@
   content: commercial
   updated: 2019-08-06
 
+- name: Doom
+  lang:
+  - C
+  license:
+  - GPL1
```

```suggestion
  - Custom
```
This release only has https://github.com/id-Software/DOOM/blob/master/linuxdoom-1.10/DOOMLIC.TXT - I can't find the GPL license in the repo

---

Sorry, you're right. This release is under their own souce license (DSL).